XRTK-Core/Packages/com.xrtk.core/Services/BoundarySystem/MixedRealityBoundarySystem.cs

```diff
@@ -624,7 +624,7 @@ public bool Contains(Vector3 location, Boundary.Type boundaryType = Boundary.Typ
             }
 
             // Handle the user teleporting (boundary moves with them).
-            location = MixedRealityToolkit.CameraSystem.CameraRig.PlayspaceTransform.InverseTransformPoint(location);
+            location = MixedRealityToolkit.CameraSystem.MainCameraRig.PlayspaceTransform.InverseTransformPoint(location);
```

Comment:
We should check if the camera system is null here and fall back to `CameraCache.Main.parent`